datahub-web-react/src/app/shared/formatNumber.ts

```diff
@@ -9,3 +9,22 @@ export function formatNumber(n) {
 export function formatNumberWithoutAbbreviation(n) {
     return n.toLocaleString();
 }
+
+export function formatBytes(bytes: number, decimals = 2): { number: number; unit: string } {
+    if (!bytes)
+        return {
+            number: 0,
+            unit: 'Bytes',
+        };
+
+    const k = 1000; // We use IEEE standards definition of units of byte, where 1000 bytes = 1kb.
+    const dm = decimals < 0 ? 0 : decimals;
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
lol at "ZB" and "YB" that's a lotta bytes

Reply:
hahaha